src/crush/CrushCompiler.cc

```diff
@@ -52,6 +52,13 @@ static void print_item_class(ostream& out, int t, CrushWrapper &crush)
     out << " class " << c;
 }
 
+static void print_class(ostream& out, int t, CrushWrapper &crush)
+{
+  const char *c = crush.get_class_name(t);
+  if (c)
+    out << " class " << c;
```

Comment:
nit: maybe this shouldn't silently print nothing if passed bad input, since the callers check